go/libkb/log_send.go

```diff
@@ -501,7 +501,7 @@ func (l *LogSendContext) LogSend(statusJSON, feedback string, sendLogs bool, num
 		// However we do use it for startup logs, since that's the only place
 		// to get them in systemd mode.
 		if l.G().Env.WantsSystemd() {
-			startLog = tailSystemdJournal(l.G().Log, []string{"keybase.service", "kbfs.service", "keybase.gui.service"}, numBytes)
+			startLog = tailSystemdJournal(l.G().Log, []string{"keybase.service", "kbfs.service", "keybase.gui.service", "keybase-redirector.service"}, numBytes)
```

Comment:
Will you change the logdump viewer on the website to have a redirector tab?  That would be nice.

Reply:
All the systemd ones just go into the startLog for now. I thought we already had a tab for redirector but maybe not. With this we'll get the redirector log for people using systemd at least.

Reply:
Made a ticket for getting redirector log on non-systemd, don't want to add to this PR